examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,532 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    from paseos.utils.set_log_level import set_log_level
</pre><p>I would suggest to simply call <code>paseos.set_log_level()</code> to avoid confusing users what all these functions are </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>